docs/docsource/operation/XDM_GUI/file_operations/download_a_file.rst

```diff
@@ -0,0 +1,26 @@
+.. _Download a File:
+
+Download a File
+===============
+
+To download a file from a selected bucket:
+
+#. Click the **Data Browser** tab open the Data Browser view.
+
+   .. image:: ../../Graphics/xdm_ui_data_browser.png
+      :width: 100%
+
+#. Select a bucket by clicking its name.
+
+   .. image:: ../../Graphics/xdm_ui_bucket_select.png
+      :width: 75%
+      
+#. Select the file to download by clicking its name.
+
+   .. image:: ../../Graphics/xdm_ui_file_select.png
+      :width: 75%
+
+#. The file download is handled by the browser.
+
+   .. image:: ../../Graphics/xdm_ui_file_download.png
```

Comment:
Is this screenshot relevant here? Because it depends on the browser used, as mentioned...

Reply:
I would argue it is. I included the browser-specific image to fill out the meaning of "handled by the browser." This graphic gives the reader context to understand that it's a browser download.